web3deployment/liquidityReserve_input.json

```diff
@@ -1,25 +1,29 @@
 {
-  "tokens": {
-    "KNC": {
+  "token": {
       "name": "KyberNetwork",
+      "symbol": "KNC",
       "decimals": 18,
-      "address": "0x4E470dc7321E84CA96FcAEDD0C8aBCebbAEB68C6"
-    }
+      "address": "0xa67bba948bd8372bc9e60b1acfafb8e2db3cc57c"
```

Comment:
is this for ropsten?
we should organize this folder.
All ropsten stuff should move to ropsten folder.
we don't want someone using this thinking its main net KNC
and this folder is not very cluttered.

Reply:
Changing it back to the ropsten KNC address. For deployment, project teams will be using replacing the token field with their own token.

Agree that we should organize the folder

Reply:
I leave it as is first, we can re-org the folder in another PR =)